.gitignore

```diff
@@ -24,4 +24,6 @@ erl_crash.dump
 
 .elixir_ls
 
+.elixir_tools
```

Comment:
I think we should remove this and the .elixir_ls line from .gitignore. These should be set per user in their own global gitignores